ClientLib/src/main/java/com/o3dr/android/client/utils/geotag/GeoTagAsyncTask.java

```diff
@@ -260,4 +271,23 @@ private static String convertLatLngToDMS(double coord) {
     protected interface GeoTagAlgorithm {
         HashMap<TLogParser.Event, File> match(List<TLogParser.Event> events, ArrayList<File> photos);
     }
+
+    private static File findNextDirName(Context context) {
+        File rootDir = new File(getSaveRootDir(context), STORE_PHOTO_DIR_NAME);
+
+        if (!rootDir.exists() && !rootDir.mkdir()) {
+            return null;
+        }
+
+        Date date = new Date();
+        String dirName = STORE_PHOTO_DIR_NAME + "_" + formatter.format(date);
```

Comment:
@chaviw be consistent in the naming scheme. The way you have it setup, if the directory doesn't exist, its name will be `%STORE_PHOTO_DIR_NAME%_%MM-DD-YY-HH%`.
If it exists, its name will be `%STORE_PHOTO_DIR_NAME%_%MM-DD-YY-HH%_%i`.

Have them all use the latter structure, with `i = 0` for case where the file doesn't exist.